demato/.htaccess

```diff
@@ -0,0 +1 @@
+Redirect 302 / https://github.com/korchiev/Demato-Ontology
```

Comment:
You need to use a full path if you are using mod_alias.  mod_redirect uses relative paths.
https://httpd.apache.org/docs/2.4/mod/mod_alias.html#redirect

Reply:
I am fairly new to this. I have created github.io page and I want this repo to redirect there? What command should I use?

Thank you so much for your help!

Reply:
I am fairly new to this. I have created github.io page and I want this repo to redirect there? What command should I use?

Thank you so much for your help!

Reply:
There's a slightly complex example here: https://github.com/perma-id/w3id.org/tree/master/example/.  As needed you may be able to strip out RewriteCond/RewriteRules lines you don't need and end up with something like the last "default" line in there.  There are many other ids you could look at for inspiration too.

Reply:
I have followed an example. Could you please verify that?

My logic was if someone accesses https://w3id.org/demato with empty URL path it will redirect to github.io page. 

Is that right?